src/main/java/org/apache/platypus/server/luceneserver/SearchHandler.java

```diff
@@ -842,9 +890,8 @@ public int docID() {
                             advance = sortedNumericDocValues.advanceExact(docID);
                             if (advance) {
                                 for (int i = 0; i < sortedNumericDocValues.docValueCount(); i++) {
-                                    long val = sortedNumericDocValues.nextValue();
-                                    compositeFieldValue.addFieldValue(FieldValue.newBuilder()
-                                            .setLongValue(val));
+                                    Long val = sortedNumericDocValues.nextValue();
```

Comment:
We can avoid autoboxing here. It's usages seem to either unbox to primitives or cast to int